eahub/config/settings.py

```diff
@@ -196,6 +196,7 @@
 
 # Django PWNED Passwords
 PWNED_VALIDATOR_FAIL_SAFE = False
+PWNED_VALIDATOR_ERROR = "Your password is not secure enough for it was determined to have been involved in a major breach before."
```

Comment:
This sentence seems garbled, somehow. Busy right now but will try to think of better wordings.

Reply:
Changed to "Your password is not secure enough". Maybe that's detailed enough?

Reply:
No, this is in a context where the user is already being told that and we're trying to explain why.

Honestly I feel like the original text was good enough here.

Reply:
I'm probably missing something, but in which sense was "the user already being told" that their password isn't secure enough? This message comes up when a user is trying to reset their password with an easily guessable password such as "TestPassword". As far as I can see they have not been told anything previous to that. And it seems that "Your password is not secure enough" is all the info the user needs to know. I agree with @michaltrzesimiech's original point that the original wording can be quite alarming and potentially misleading.

That said, if you feel strongly about this and @michaltrzesimiech on the other hand not too much, let's stick with the original wording.

Reply:
Ah, sorry, I wasn't clear. The issue is that in this context the user is shown a bulleted list of which specific password rules their password fails. "Not secure enough" doesn't make sense in that context.

Reply:
How about we leave it almost exactly as it was, but add "in the past": makes clear that the user has not caused a security breach:
"Your password was determined to have been involved in a major security breach in the past"

Reply:
Maybe we could do better than that by actually linking to the Pwned Passwords database so that the user has some clue of what's going on and where we got their old passwords from.

Reply:
add link to URL.